guides/04_Quality_Bandwidth_Connectivity.md

```diff
@@ -25,13 +25,6 @@ The Amazon Chime SDK for JavaScript produces several kinds of events on the [Aud
 
 *For the browser support columns below, "All" refers to the browsers officially supported by the Chime SDK.*
 
-### Events for monitoring local attendee uplink
```

Comment:
Should we update the `metricsDidReceive` event under "Event" table here?

Reply:
I don't much is needed to be updated, but I did add that you can now also get Audio Jitter